### PR TITLE
Enable test_send_tokens_without_chain_id test

### PR DESCRIPTION
**Pull request by lyubenkov**

### CODE REVIEW

**Comment by lyubenkov** [comment]
Passed build https://github.com/neonlabsorg/neon-tests/actions/runs/8832927991

File: integration/tests/economy/test_economics.py [hunk 1]
```diff
@@ -124,7 +123,12 @@ def test_send_tokens_without_chain_id(
         )
         instruction_tx.pop("chainId")
 
-        w3_client.send_transaction(account_with_all_tokens, instruction_tx)
+        try:
+            w3_client.send_transaction(account_with_all_tokens, instruction_tx)
```
Comment (kristinaNikolaevaa) [inline integration/tests/economy/test_economics.py]: let's split if for 2 tests: first one to check economy for transaction without chain_id (only for neon chain) and second move from economy tests to basic with error message check (for some another chain exept neon)

**Review by kristinaNikolaevaa — CHANGES_REQUESTED** [review]

File: integration/tests/economy/test_economics.py (hunk 1 above)
Comment (lyubenkov) [reply integration/tests/economy/test_economics.py]: Done

File: integration/tests/basic/test_failed_transactions.py [hunk 2]
```diff
@@ -24,3 +25,19 @@ def test_bump_allocator_out_of_memory_expected_error(self):
             assert resp["status"] == 0
         except ValueError as exc:
             assert "Error: memory allocation failed, out of memory." in exc.args[0]["message"]
+
+    def test_send_non_neon_token_without_chain_id(self, account_with_all_tokens, web3_client_sol, sol_price, operator):
```
Comment (kristinaNikolaevaa) [inline integration/tests/basic/test_failed_transactions.py]: Please, move it to test_payment_in_different_tokens.py, add to md file (where tests are described), add multipletokens mark, and change try-exept block to pytest.raise

**Review by kristinaNikolaevaa — CHANGES_REQUESTED** [review]

File: integration/tests/basic/test_failed_transactions.py (hunk 2 above)
Comment (lyubenkov) [reply integration/tests/basic/test_failed_transactions.py]: Done

**Review by kristinaNikolaevaa — APPROVED** [review]

**Pull request merged**